### PR TITLE
add dbt server repo

### PR DESCRIPTION
**Pull request by mirnawong1**
adding public dbt server repo for sl docs

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by mirnawong1** [comment]
hi @callum-mcdata can i ask you to review this for technical accuracy? once you have a look, I'll then ask the docs team for a review. many thanks!

File: website/docs/docs/use-dbt-semantic-layer/quickstart-semantic-layer.md [hunk 1]
```diff
@@ -318,3 +318,4 @@ Are you ready to define your own metrics and bring consistency to data consumers
 - [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-semantic-layer) to learn about the dbt Semantic Layer
 - [Understanding the components of the dbt Semantic Layer](https://docs.getdbt.com/blog/understanding-the-components-of-the-dbt-semantic-layer) blog post to see further examples
 - [Integrated partner tools](https://www.getdbt.com/product/semantic-layer-integrations) for info on the different integration partners and their documentation
+- [dbt Server repo](https://github.com/dbt-labs/dbt-server), which allows you to compile metric queries per environment using dbt project code
```
Comment (callum-mcdata) [inline website/docs/docs/use-dbt-semantic-layer/quickstart-semantic-layer.md]: I'd change the language here to something along the lines of:

dbt Server wraps dbt Core in a persistent server that is responsible for handling RESTful API requests for dbt operations.

File: website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md [hunk 2]
```diff
@@ -51,3 +51,4 @@ Before you set up the dbt Semantic Layer, make sure you meet the following:
 - [Integrated partner tools](https://www.getdbt.com/product/semantic-layer-integrations) for info on the different integration partners and their documentation
 - [Product architecture](/docs/use-dbt-semantic-layer/dbt-semantic-layer#product-architecture) page for more information on plan availability
 - [dbt metrics](/docs/build/metrics) for in-depth detail on attributes, properties, filters, and how to define and query metrics
+- [dbt Server repo](https://github.com/dbt-labs/dbt-server), which allows you to compile metric queries per environment using dbt project code
```
Comment (callum-mcdata) [inline website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md]: same as above

**Comment by mirnawong1** [comment]
made those changes now @callum-mcdata !

**Review by callum-mcdata — APPROVED** [review]
LGTM!

**Pull request merged**